internal/controller/cilium_flow_manager.go

```diff
@@ -0,0 +1,228 @@
+package controller
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/cilium/cilium/api/v1/flow"
+	observer "github.com/cilium/cilium/api/v1/observer"
+	pb "github.com/illumio/cloud-operator/api/illumio/cloud/k8scluster/v1"
+	"go.uber.org/zap"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+// FlowManager holds a logger and Cilium client
+type FlowManager struct {
+	logger *zap.SugaredLogger
+	client observer.ObserverClient
+}
+
+// discoverHubbleRelayAddress uses a kubernetes clientset in order to discover the address of hubble-relay within kube-system.
+func discoverHubbleRelayAddress(ctx context.Context, logger *zap.SugaredLogger, clientset kubernetes.Interface) (string, error) {
+	service, err := clientset.CoreV1().Services("kube-system").Get(ctx, "hubble-relay", metav1.GetOptions{})
```

Comment:
Make the namespace configurable. It is configurable in the Cilium Helm chart, so it may not always be `kube-system`.
https://github.com/cilium/cilium/blob/f8cccace805b203cc3e0c5fd5c4dab6aa7d88b9a/install/kubernetes/cilium/templates/hubble-relay/service.yaml#L6

Reply:
Done